src/api/views.py

```diff
@@ -83,58 +84,37 @@ def validate(request):
+            elif response:
+                result = response
+            else:
+                result = message
+            
+            if httpstatus == 200:
+                returnstatus = status.HTTP_200_OK
+            elif httpstatus == 400:
+                returnstatus = status.HTTP_400_BAD_REQUEST
+            else:
+                returnstatus = status.HTTP_404_NOT_FOUND
+            
+            query = ValidateFileUpload.objects.create(owner=request.user, file=request.data.get('file'))
+            query.result = result
+            query.status = httpstatus
             uploaded_file = str(query.file)
             uploaded_file_path = str(query.file.path)
-            try :
-                if request.FILES["file"]:
-                    formatstr = request.POST["format"]
-                    serFileTypeEnum = jpype.JClass("org.spdx.tools.SpdxToolsHelper$SerFileType")
-                    fileformat = serFileTypeEnum.valueOf(formatstr)
-                    """ Call the java function with parameter"""
-                    retval = verifyclass.verify(uploaded_file_path, fileformat)
-                    if (len(retval) > 0):
-                        result = "The following error(s)/warning(s) were raised: " + str(retval)
-                        returnstatus = status.HTTP_400_BAD_REQUEST
-                        httpstatus = 400
-                        jpype.detachThreadFromJVM()
-                    else :
-                        result = "This SPDX Document is valid."
-                        returnstatus = status.HTTP_201_CREATED
-                        httpstatus = 201
-                        jpype.detachThreadFromJVM()
-                else :
-                    result = "File Not Uploaded"
-                    returnstatus = status.HTTP_400_BAD_REQUEST
-                    httpstatus = 400
-                    jpype.detachThreadFromJVM()
-            except jpype.JException as ex :
-                """ Error raised by verifyclass.verify without exiting the application"""
-                result = jpype.JException.message(ex) #+ "This SPDX Document is not a valid RDF/XML or tag/value format"
-                returnstatus = status.HTTP_400_BAD_REQUEST
-                httpstatus = 400
-                jpype.detachThreadFromJVM()
-            except :
-                """ Other errors raised"""
-                result = format_exc()
-                returnstatus = status.HTTP_400_BAD_REQUEST
-                httpstatus = 400
-                jpype.detachThreadFromJVM()
-            query.result=result
-            query.status=httpstatus
-            ValidateFileUpload.objects.filter(file=uploaded_file).update(result=result,status=httpstatus)
+            ValidateFileUpload.objects.filter(file=uploaded_file).update(result=result, status=httpstatus)
             serial = ValidateSerializerReturn(instance=query)
             return Response(
                 serial.data, status=returnstatus
```

Comment:
Can we other functions apart from views functions like `extensionGiven` , `getFileFormat`, etc. to a utils.py file? Basically this file should contain only core logic, other business logic function should be moved to utils function

Reply:
Sure. I will do that.